pkg/operator/ceph/object/spec.go

```diff
@@ -569,7 +570,7 @@ func addPortToEndpoint(endpoints *v1.Endpoints, name string, port int32) {
 }
 
 func getLabels(name, namespace string, includeNewLabels bool) map[string]string {
-	labels := controller.CephDaemonAppLabels(AppName, namespace, "rgw", name, includeNewLabels)
+	labels := controller.CephDaemonAppLabels(AppName, namespace, config.RgwType, name, "rook-operator", "cephobjectstores.ceph.rook.io", includeNewLabels)
```

Comment:
The object store name seems correct for the parent name.
```suggestion
	labels := controller.CephDaemonAppLabels(AppName, namespace, config.RgwType, name, name, "cephobjectstores.ceph.rook.io", includeNewLabels)
```